src/ImageSharp/Formats/Jpg/JpegEncoderCore.cs

```diff
@@ -847,23 +849,25 @@ private void WriteDefineHuffmanTables(int componentCount)
             where TColor : struct, IPackedPixel<TPacked>
             where TPacked : struct
         {
-            Block b = new Block();
-            Block cb = new Block();
-            Block cr = new Block();
-            
+            Block b = Block.Create();
+            Block cb = Block.Create();
+            Block cr = Block.Create();
             // ReSharper disable once InconsistentNaming
             int prevDCY = 0, prevDCCb = 0, prevDCCr = 0;
 
             for (int y = 0; y < pixels.Height; y += 8)
```

Comment:
I think we need `Block8x8F` here. `JpegEncoderCore` should be refactored to use it instead of the old `Block` class.